source/codegen/metadata/nixnet/functions.py

```diff
@@ -166,28 +172,32 @@
                 'direction': 'out',
                 'name': 'valueBuffer',
                 'size': {
-                    'mechanism': 'passed-in',
-                    'value': 'sizeOfValueBuffer'
+                    'mechanism': 'custom-code',
```

Comment:
This is actually `passed-in` now :)

Same for `timeStampBuffer` param below, and in other APIs.